docs/VerifPlans/Simulation/Zc/RV32Zc_Extension_Instructions.json

```diff
@@ -32,7 +32,7 @@
         "Pass/Fail Criteria": "Check against ISS",
         "Test Type": "Constrained-Random",
         "Coverage Method": "Functional Coverage",
-        "Link to Coverage": "Imperas generated coverage, hierarchy path:\nuvmt_cv32e40s_tb.imperas_dv.idv_trace2cov.cov_1.obj_[instr]",
+        "Link to Coverage": "Imperas generated coverage, hierarchy path:\nuvmt_cv32e40x_tb.imperas_dv.idv_trace2cov.cov_1.obj_[instr]",
```

Comment:
Is this a "blanket" substitusion of S->X in all vplans?

That seems dangerous, because some texts say "this is valid for S and not X" and now it will be scrambled.

Reply:
A possible way to deal with this is to regen all csv/json files.

E.g. something like: `find docs | grep xlsx | xargs -n1 ../bin/xlsx2csv`.

Reply:
FIXED.